app/views/invitation/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_tag do%>
```

Comment:
We should probably add a header/instructions for this form

Reply:
Yes! Will do